src/main/java/com/github/nicholasmoser/gnt4/seq/SeqKingHtml.java

```diff
@@ -77,15 +82,53 @@ private static ContainerTag getBody(List<Opcode> opcodes) {
     return body;
   }
 
+  public static boolean isFunctionEnd(int index, List<Opcode> opcodes) {
```

Comment:
The PR shows this function as having been changed, but I know for a fact this is already in master. I actually notice that for a lot of the changes in this PR. I think it may actually be an issue with Github though, because when I looked around the internet I found a potential solution: https://stackoverflow.com/a/46782679

```"Here's a good workaround. Use the Edit button when viewing the PR in GitHub to change the base branch to something other than master. Then switch it back to master and it will now correctly show only the changes from the most recent commits."```

Hopefully if you try doing that it'll clean up the PR and only show the actual changes you made.

Reply:
Yep that fixed it!